### DISCUSSION TITLE
Support for comma-separated values in condition builder

### DISCUSSION THREAD

**Discussion by low** [craftcms/cms#16121] [Ideas] [answered]
When filtering elements in the CP (or elsewhere where you can define conditions), I'd love to be able to use the `IN` / `NOT IN` operators for Text and Number based conditions, that take a comma-separated list of values.

In my specific use case, I have a specific set of element IDs that I want to export using the Export utility in the element listings. Having the option to filter by `ID IN 1,2,3` would be very helpful.

**Comment by brandonkelly** [comment, marked as answer]
Just added support for specifying multiple elements for Craft 5.6 (6bf0270eae04ca5ed22bcad9fb6e294d04009dfd) 🎉

**Reply by brandonkelly** [reply]
Craft 5.6.0 is out with that change 🎉
